hopp/simulation/technologies/resource/nsrdb_data.py

```diff
@@ -0,0 +1,226 @@
+            # NOTE: solar_resource.py code references solar_zenith_angle and RH = relative_humidity but I couldn't find them actually being utilized. Captured them below just in case.
+            self.year_arr = f.time_index.year.values[1::2]
+            self.month_arr = f.time_index.month.values[1::2]
+            self.day_arr = f.time_index.day.values[1::2]
+            self.hour_arr = f.time_index.hour.values[1::2]
+            self.minute_arr = f.time_index.minute.values[1::2]
+            self.dni_arr = SAMResource.roll_timeseries((f['dni', :, site_gid][1::2]), self.time_zone, 1)
+            self.dhi_arr = SAMResource.roll_timeseries((f['dhi', :, site_gid][1::2]), self.time_zone, 1)
+            self.ghi_arr = SAMResource.roll_timeseries((f['ghi', :, site_gid][1::2]), self.time_zone, 1)
+            self.wspd_arr = SAMResource.roll_timeseries((f['wind_speed', :, site_gid][1::2]), self.time_zone, 1)
+            self.tdry_arr = SAMResource.roll_timeseries((f['air_temperature', :, site_gid][1::2]), self.time_zone, 1)
+            # self.relative_humidity_arr = SAMResource.roll_timeseries((f['relative_humidity', :, site_gid][1::2]), self.time_zone, 1)
+            # self.solar_zenith_arr = SAMResource.roll_timeseries((f['solar_zenith_angle', :, site_gid][1::2]), self.time_zone, 1)
+            self.pres_arr = SAMResource.roll_timeseries((f['surface_pressure', :, site_gid][1::2]), self.time_zone, 1)
+            self.tdew_arr = SAMResource.roll_timeseries((f['dew_point', :, site_gid][1::2]), self.time_zone, 1)
+
+            self.site_gid = site_gid
+            
+
+    def format_data(self):
+        # Remove data from feb29 on leap years
+        if (self.year % 4) == 0:
+            feb29 = np.arange(1416,1440)
+            self.year_arr = np.delete(self.year_arr, feb29)
+            self.month_arr = np.delete(self.month_arr, feb29)
+            self.day_arr = np.delete(self.day_arr, feb29)
+            self.hour_arr = np.delete(self.hour_arr, feb29)
+            self.minute_arr = np.delete(self.minute_arr, feb29)
+            self.dni_arr = np.delete(self.dni_arr, feb29)
+            self.dhi_arr = np.delete(self.dhi_arr, feb29)
+            self.ghi_arr = np.delete(self.ghi_arr, feb29)
+            self.wspd_arr = np.delete(self.wspd_arr, feb29)
+            self.tdry_arr = np.delete(self.tdry_arr, feb29)
+            # self.relative_humidity_arr = np.delete(self.relative_humidity_arr, feb29)
+            # self.solar_zenith_arr = np.delete(self.solar_zenith_arr, feb29)
+            self.pres_arr = np.delete(self.pres_arr, feb29)
+            self.tdew_arr = np.delete(self.tdew_arr, feb29)
+
+        # round to desired precision and convert to desired data type
+        # NOTE: unsure if SAM/PySAM is sensitive to data types and decimal precision. 
+        # If not sensitive, can remove .astype() and round() to increase computational efficiency
```

Comment:
Is there a good way to find out? It would be nice to have it settled prior to merging, but if this is pretty small potatoes, I also get it.